climada/util/coordinates.py

```diff
@@ -2655,26 +2658,28 @@ def set_df_geometry_points(df_val, scheduler=None, crs=None):
         try:
             crs = df_val.geometry.crs
         except AttributeError:
-            crs = None
+            pass
```

Comment:
Any GeoDataFrame has the attribute [`.crs`](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoDataFrame.crs.html#geopandas.GeoDataFrame.crs). If it is not set, the attribute already returns None. So this can simply be
```python
crs = df_val.crs if crs is None else crs  # crs might now still be None
```

Reply:
👍